packages/lit-html/src/lit-html.ts

```diff
@@ -366,17 +371,21 @@ class Template {
             regex = comment2Regex;
           } else if (match[2] !== undefined) {
             if (rawTextElement.test(match[2])) {
-              regex = new RegExp(`<\/${match[2]}`, 'g');
-            } else {
-              regex = tagRegex;
+              // Record if we encounter a raw-text element. We'll switch to
```

Comment:
I understand that the matches are not named to save code, but I suggest having comments above each section clearly labelling the meaning of each match.

Reply:
Added consts which should be inlined by Terser. We'll need to keep an eye on the output though.